Modules/Core/Common/test/itkNeighborhoodIteratorTest.cxx

```diff
@@ -83,7 +83,7 @@ itkNeighborhoodIteratorTest(int, char *[])
 
   it3.Print(std::cout);
   unsigned x, y, i;
-  for (y = 0, i = 0; y < 5; y++)
+  for (y = 0, i = 0; y < 5; ++y)
   {
     for (x = 0; x < 5; x++, i++)
```

Comment:
FYI, here post-increment was not detected by regex, probably (and several below)

Reply:
Probably all multi-variable increments are missed.

Reply:
Good catch @issakomi Do you have a suggestion how to catch and fix such cases by regex? 

Obviously we must be 100% sure that the semantics don't change by the regex!

Reply:
Sorry, not yet sure how to automate this in reliable way. Looks like in Core/Common multi-variable _for_  with _postfix_ increment are below, at least with "_for_" and "_,_" at a single line

```
./test/itkNeighborhoodIteratorTest.cxx:    for (x = 0; x < 5; x++, i++)
./test/itkNeighborhoodIteratorTest.cxx:    for (x = 0; x < 5; x++, i++)
./test/itkNeighborhoodIteratorTest.cxx:    for (x = 0; x < 5; x++, i++)
./test/itkNeighborhoodIteratorTest.cxx:    for (x = 0; x < 5; x++, i++)
./test/itkNeighborhoodIteratorTest.cxx:    for (x = 0; x < 5; x++, i++)
./test/itkNeighborhoodIteratorTest.cxx:    for (x = 0; x < 5; x++, i++)
./test/itkNeighborhoodIteratorTest.cxx:    for (x = 0; x < 5; x++, i++)
./test/itkNeighborhoodIteratorTest.cxx:    for (x = 0; x < 5; x++, i++)
./test/itkNeighborhoodIteratorTest.cxx:    for (x = 0; x < 5; x++, i++)
./test/itkObjectFactoryTest.cxx:    for (std::list<std::string>::const_iterator o = overrides.begin(); o != overrides.end(); ++o, ++n, ++d, e++)
./test/itkPriorityQueueTest.cxx:  for (; it != sequence.end(); ++it, i++)
./test/itkObjectFactoryTest2.cxx:      for (std::list<std::string>::const_iterator o = overrides.begin(); o != overrides.end(); ++o, ++n, ++d, e++)
./include/itkNeighborhoodIterator.hxx:    for (N_it = N.Begin(), this_it = this->Begin(); this_it < _end; this_it++, N_it++)
./include/itkNeighborhoodIterator.hxx:    for (N_it = N.Begin(), this_it = this->Begin(); this_it < _end; this_it++, N_it++)
./include/itkNeighborhoodIterator.hxx:    for (N_it = N.Begin(), this_it = this->Begin(); this_it < _end; N_it++, this_it++)
```

Reply:
We don't have to change every instance now. Getting most of them is sufficient improvement.

Reply:
Please check my latest force-push 😄  https://github.com/InsightSoftwareConsortium/ITK/compare/ff9e1a7584564ff6a54b5191277847793ffe9fe4..d7eea8a76120648be23474c379894ef0338878f0